tests/jax/core_system_components/trainer_test.py

```diff
@@ -13,4 +13,57 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Tests for trainer class for Jax-based Mava systems"""
+from types import SimpleNamespace
+from typing import List
+
+import pytest
+
+from mava.callbacks import Callback
+from mava.systems.jax.trainer import Trainer
+from tests.jax.hook_order_tracking import HookOrderTracking
+
+
+class TestTrainer(HookOrderTracking, Trainer):
+    def __init__(
+        self,
+        config: SimpleNamespace,
```

Comment:
At some point, we still need to go rename a bunch of these, since in a bunch of the core components, they call this `config` instead of `store`. Will make a separate PR for this though.